### DISCUSSION TITLE
"fake" drop down to overlap sibling elements

### DISCUSSION THREAD

**Discussion by nora-kauczor** [neuefische/questions#842] [Web]
- We're styling our [FilteredRecipe component](https://github.com/Larissartoricader/Capstone/blob/style-filter/components/FilteredRecipes.js)
- [These are its styled components](https://github.com/Larissartoricader/Capstone/blob/style-filter/components/FilteredRecipes.styles.js)
- `SearchfieldAndDropDown` consists of an input field and divs that are only being shown conditionally.
- **We need those divs (we're calling this a fake drop down) to overlap the elements underneath it**
- Therefore, we need to make sure that its container, the element `SearchBox`, overlaps its siblings `Selection` and [RecipeList](https://github.com/Larissartoricader/Capstone/blob/style-filter/components/RecipeList.js)
- If we give `SearchBox` a z-index, e.g. of 3, it overlaps the [Header ](https://github.com/Larissartoricader/Capstone/blob/style-filter/components/Header.js) when scrolling (event though the header's z-index is 1). This is a behaviour we don't want.

Thanks a lot in advance.

**Comment by F-Kirchhoff** [comment]
Hey Nora,
wenn ihr folgenden aufbau in eurem Layout habt: 
```
<header />
<content />
<navigation />
```
Könnt ihr dem content einen eigenen z-index geben z.b. 1. Der Header und Nav können dann einen z-index von 2 bekommen. Dadurch sollte verhindert werden, dass die Searchbox über dem header dargestellt wird. Ihr müsst dafür sichergehen, dass der Content zwischen header und navigation immer in einem element, z.B. einem `main` element, platziert ist. Dieses Element kann dann den entsprechenden z-index bekommen.
